### ISSUE TITLE
Default Deviations Missing for several Clinical Biospecimen Tubes

### ISSUE THREAD

**Issue by FrogGirl1123** [episphere/connect#579]
See tables below for details.  The center column indicates the type of tube under the Object ID and the numbers are a count of the number of deviation defaults for each tube type.  Only the Serum Separator tubes have all the necessary deviation defaults excluding the one specific to mouthwash and the one specific to urine.  Please ignore the _# associated with the fields in column A and the CIDs in column C, they are a result of manual flattening.
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/GERLAN~1/AppData/Local/Temp/1/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/GERLAN~1/AppData/Local/Temp/1/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Connect_ID |   | Connect_ID
-- | -- | --
BioSpm_Visit_v1r0 |   | d_331584571
RcrtES_Site_v1r0 |   | d_827220437
Collection ID |   | d_820476880
DB urine accession number |   | d_928693120
Bld Acc Manul |   | d_148996099
Blood Accession ID |   | d_646899796
Collection ID Manually Entered |   | d_387108065
Collection Setting |   | d_650516960
Date/Time scanned at Regional   Lab |   | d_915838974
Additional notes on Collection |   | d_338570265
Object ID | Biohazzard blood/urine bag | d_825582494
Object Collected |   | d_593843561
Collection Entry Finalized |   | d_410912345
Object ID_1 | ACD Tube 1 | d_825582494_1
Deviation |   | d_678857215
Discard flag |   | d_762124027
Object Collected_1 |   | d_593843561_1
Deviation - Not found | 1 | d_982885431
Deviation - Unexpected tube   size or type | 2 | d_777486216
Deviation - Discard | 3 | d_810960823
Deviation- Low volume | 4 | d_728366619
Deviation- Other | 5 | d_453343022
Deviation- Improper   temperature - too high | 6 | d_690540566
Deviation- Hemolyzed | 7 | d_242307474
Deviation- Broken | 8 | d_472864016
Deviation- Leaked/spilled | 9 | d_757246707
Deviation - Mislabeled-   discard | 10 | d_684617815
Deviation - Mislabeled-   Resolved | 11 | d_283900611
Deviation-Improper temperature   - too low | 12 | d_550088682
Deviation - Outside of tube contaminated | 13 | d_313097539
Date Received - Site Shipment |   | d_926457119
Deviation Details |   | d_536710547
Object ID_2 | Urine Tube 1 | d_825582494_2
Deviation_1 |   | d_678857215_1
Discard flag_1 |   | d_762124027_1
Object Collected_2 |   | d_593843561_2
Deviation- Improper   temperature - too high_1 | 1 | d_690540566_1
Deviation- Broken_1 | 2 | d_472864016_1
Deviation- Insufficient volume-urine | 3 | d_956345366
Deviation- Leaked/spilled_1 | 4 | d_757246707_1
Deviation - Not found_1 | 5 | d_982885431_1
Deviation - Mislabeled-   discard_1 | 6 | d_684617815_1
Deviation - Mislabeled-   Resolved_1 | 7 | d_283900611_1
Deviation-Improper temperature   - too low_1 | 8 | d_550088682_1
Deviation- Low volume_1 | 9 | d_728366619_1
Deviation - Outside of tube   contaminated_1 | 10 | d_313097539_1
Deviation- Other_1 | 11 | d_453343022_1
Date Received - Site Shipment_1 |   | d_926457119_1
Deviation Details_1 |   | d_536710547_1
Date/Time Collection Entry   Finalized |   | d_556788178
Object ID_3 | Serum Separator Tube 1 | d_825582494_3
Deviation_2 |   | d_678857215_2
Discard flag_2 |   | d_762124027_2
Object Collected_3 |   | d_593843561_3
Deviation - Not found_2 | 1 | d_982885431_2
Deviation- Centrifuge -   improper time of spinning (too short) | 2 | d_937362785
Deviation- Centrifuge -   improper time of spinning (too long) | 3 | d_654002184
Deviation- Centrifuge -   clotted for longer than 2 hours | 4 | d_912088602
Deviation - Unexpected tube   size or type_1 | 5 | d_777486216_1
Deviation - Discard_1 | 6 | d_810960823_1
Deviation- Low volume_2 | 7 | d_728366619_2
Deviation- Centrifuge -   Improper speed-high | 8 | d_861162895
Deviation- Other_2 | 9 | d_453343022_2
Deviation- Improper   temperature - too high_2 | 10 | d_690540566_2
Deviation- Hemolyzed_1 | 11 | d_242307474_1
Deviation- Broken_2 | 12 | d_472864016_2
Deviation- Leaked/spilled_2 | 13 | d_757246707_2
Deviation - Mislabeled-   discard_2 | 14 | d_684617815_2
Deviation- Failed/broken gel   layer | 15 | d_635875253
Deviation - Mislabeled-   Resolved_2 | 16 | d_283900611_2
Deviation- Centrifuge -   Improper speed/force setting used (too low) | 17 | d_561005927
Deviation- Centrifuge - did   not clot at least 30 minutes | 18 | d_102695484
Deviation-Improper temperature   - too low_2 | 19 | d_550088682_2
Deviation - Outside of tube   contaminated_2 | 20 | d_313097539_2
Date Received - Site   Shipment_2 |   | d_926457119_2
Deviation Details_2 |   | d_536710547_2
Date Received - Site   Shipment_3 |   | d_926457119_3
Object ID_4 | Serum Separator Tube 2 | d_825582494_4
Deviation_3 |   | d_678857215_3
Discard flag_3 |   | d_762124027_3
Object Collected_4 |   | d_593843561_4
Deviation - Not found_3 | 1 | d_982885431_3
Deviation- Centrifuge -   improper time of spinning (too short)_1 | 2 | d_937362785_1
Deviation- Centrifuge -   improper time of spinning (too long)_1 | 3 | d_654002184_1
Deviation- Centrifuge -   clotted for longer than 2 hours_1 | 4 | d_912088602_1
Deviation - Unexpected tube   size or type_2 | 5 | d_777486216_2
Deviation - Discard_2 | 6 | d_810960823_2
Deviation- Low volume_3 | 7 | d_728366619_3
Deviation- Centrifuge -   Improper speed-high_1 | 8 | d_861162895_1
Deviation- Other_3 | 9 | d_453343022_3
Deviation- Improper   temperature - too high_3 | 10 | d_690540566_3
Deviation- Hemolyzed_2 | 11 | d_242307474_2
Deviation- Broken_3 | 12 | d_472864016_3
Deviation- Leaked/spilled_3 | 13 | d_757246707_3
Deviation - Mislabeled-   discard_3 | 14 | d_684617815_3
Deviation- Failed/broken gel   layer_1 | 15 | d_635875253_1
Deviation - Mislabeled-   Resolved_3 | 16 | d_283900611_3
Deviation- Centrifuge -   Improper speed/force setting used (too low)_1 | 17 | d_561005927_1
Deviation- Centrifuge - did not clot at   least 30 minutes_1 | 18 | d_102695484_1
Deviation-Improper temperature   - too low_3 | 19 | d_550088682_3
Deviation - Outside of tube   contaminated_3 | 20 | d_313097539_3
Date Received - Site   Shipment_4 |   | d_926457119_4
Object ID_5 | Serum Separator Tube 3 | d_825582494_5
Deviation_4 |   | d_678857215_4
Discard flag_4 |   | d_762124027_4
Object Collected_5 |   | d_593843561_5
Deviation - Not found_4 | 1 | d_982885431_4
Deviation- Centrifuge -   improper time of spinning (too short)_2 | 2 | d_937362785_2
Deviation- Centrifuge -   improper time of spinning (too long)_2 | 3 | d_654002184_2
Deviation- Centrifuge -   clotted for longer than 2 hours_2 | 4 | d_912088602_2
Deviation - Unexpected tube   size or type_3 | 5 | d_777486216_3
Deviation- Low volume_4 | 6 | d_728366619_4
Deviation - Discard_3 | 7 | d_810960823_3
Deviation- Centrifuge -   Improper speed-high_2 | 8 | d_861162895_2
Deviation- Other_4 | 9 | d_453343022_4
Deviation- Hemolyzed_3 | 10 | d_242307474_3
Deviation- Improper   temperature - too high_4 | 11 | d_690540566_4
Deviation- Broken_4 | 12 | d_472864016_4
Deviation- Leaked/spilled_4 | 13 | d_757246707_4
Deviation - Mislabeled-   discard_4 | 14 | d_684617815_4
Deviation - Mislabeled-   Resolved_4 | 15 | d_283900611_4
Deviation- Failed/broken gel   layer_2 | 16 | d_635875253_2
Deviation- Centrifuge -   Improper speed/force setting used (too low)_2 | 17 | d_561005927_2
Deviation- Centrifuge - did   not clot at least 30 minutes_2 | 18 | d_102695484_2
Deviation-Improper temperature   - too low_4 | 19 | d_550088682_4
Deviation - Outside of tube   contaminated_4 | 20 | d_313097539_4
Date Received - Site   Shipment_5 |   | d_926457119_5
Object ID_6 | Serum Separator Tube 4 | d_825582494_6
Deviation_5 |   | d_678857215_5
Discard flag_5 |   | d_762124027_5
Object Collected_6 |   | d_593843561_6
Deviation - Not found_5 | 1 | d_982885431_5
Deviation- Centrifuge -   improper time of spinning (too short)_3 | 2 | d_937362785_3
Deviation- Centrifuge -   improper time of spinning (too long)_3 | 3 | d_654002184_3
Deviation- Centrifuge -   clotted for longer than 2 hours_3 | 4 | d_912088602_3
Deviation - Unexpected tube   size or type_4 | 5 | d_777486216_4
Deviation- Centrifuge -   Improper speed-high_3 | 6 | d_861162895_3
Deviation- Low volume_5 | 7 | d_728366619_5
Deviation - Discard_4 | 8 | d_810960823_4
Deviation- Other_5 | 9 | d_453343022_5
Deviation- Improper   temperature - too high_5 | 10 | d_690540566_5
Deviation- Hemolyzed_4 | 11 | d_242307474_4
Deviation- Broken_5 | 12 | d_472864016_5
Deviation- Leaked/spilled_5 | 13 | d_757246707_5
Deviation - Mislabeled-   discard_5 | 14 | d_684617815_5
Deviation - Mislabeled-   Resolved_5 | 15 | d_283900611_5
Deviation- Failed/broken gel   layer_3 | 16 | d_635875253_3
Deviation- Centrifuge -   Improper speed/force setting used (too low)_3 | 17 | d_561005927_3
Deviation- Centrifuge - did   not clot at least 30 minutes_3 | 18 | d_102695484_3
Deviation-Improper temperature   - too low_5 | 19 | d_550088682_5
Deviation - Outside of tube   contaminated_5 | 20 | d_313097539_5
Date Received - Site   Shipment_6 |   | d_926457119_6
Object ID_7 | Serum Separator Tube 5 | d_825582494_7
Deviation_6 |   | d_678857215_6
Discard flag_6 |   | d_762124027_6
Object Collected_7 |   | d_593843561_7
Deviation - Not found_6 | 1 | d_982885431_6
Deviation- Centrifuge -   improper time of spinning (too short)_4 | 2 | d_937362785_4
Deviation- Centrifuge -   improper time of spinning (too long)_4 | 3 | d_654002184_4
Deviation- Centrifuge -   clotted for longer than 2 hours_4 | 4 | d_912088602_4
Deviation - Unexpected tube   size or type_5 | 5 | d_777486216_5
Deviation- Low volume_6 | 6 | d_728366619_6
Deviation - Discard_5 | 7 | d_810960823_5
Deviation- Centrifuge -   Improper speed-high_4 | 8 | d_861162895_4
Deviation- Other_6 | 9 | d_453343022_6
Deviation- Improper   temperature - too high_6 | 10 | d_690540566_6
Deviation- Hemolyzed_5 | 11 | d_242307474_5
Deviation- Broken_6 | 12 | d_472864016_6
Deviation- Leaked/spilled_6 | 13 | d_757246707_6
Deviation - Mislabeled-   discard_6 | 14 | d_684617815_6
Deviation- Failed/broken gel   layer_4 | 15 | d_635875253_4
Deviation - Mislabeled-   Resolved_6 | 16 | d_283900611_6
Deviation- Centrifuge -   Improper speed/force setting used (too low)_4 | 17 | d_561005927_4
Deviation- Centrifuge - did   not clot at least 30 minutes_4 | 18 | d_102695484_4
Deviation-Improper temperature   - too low_6 | 19 | d_550088682_6
Deviation - Outside of tube   contaminated_6 | 20 | d_313097539_6
Date Received - Site   Shipment_7 |   | d_926457119_7
Object ID_8 | Heparin Tube 1 | d_825582494_8
Deviation_7 |   | d_678857215_7
Discard flag_7 |   | d_762124027_7
Object Collected_8 |   | d_593843561_8
Deviation - Not found_7 | 1 | d_982885431_7
Deviation - Unexpected tube   size or type_6 | 2 | d_777486216_6
Deviation - Discard_6 | 3 | d_810960823_6
Deviation- Low volume_7 | 4 | d_728366619_7
Deviation- Other_7 | 5 | d_453343022_7
Deviation- Improper   temperature - too high_7 | 6 | d_690540566_7
Deviation- Hemolyzed_6 | 7 | d_242307474_6
Deviation- Broken_7 | 8 | d_472864016_7
Deviation- Leaked/spilled_7 | 9 | d_757246707_7
Deviation - Mislabeled-   discard_7 | 10 | d_684617815_7
Deviation - Mislabeled-   Resolved_7 | 11 | d_283900611_7
Deviation-Improper temperature   - too low_7 | 12 | d_550088682_7
Deviation - Outside of tube   contaminated_7 | 13 | d_313097539_7
Date Received - Site   Shipment_8 |   | d_926457119_8
Deviation Details_3 |   | d_536710547_3
Object ID_9 | Heparin Tube 2 | d_825582494_9
Deviation_8 |   | d_678857215_8
Discard flag_8 |   | d_762124027_8
Object Collected_9 |   | d_593843561_9
Deviation - Not found_8 | 1 | d_982885431_8
Deviation - Unexpected tube   size or type_7 | 2 | d_777486216_7
Deviation - Discard_7 | 3 | d_810960823_7
Deviation- Low volume_8 | 4 | d_728366619_8
Deviation- Other_8 | 5 | d_453343022_8
Deviation- Hemolyzed_7 | 6 | d_242307474_7
Deviation- Improper   temperature - too high_8 | 7 | d_690540566_8
Deviation- Broken_8 | 8 | d_472864016_8
Deviation- Leaked/spilled_8 | 9 | d_757246707_8
Deviation - Mislabeled-   discard_8 | 10 | d_684617815_8
Deviation - Mislabeled-   Resolved_8 | 11 | d_283900611_8
Deviation-Improper temperature   - too low_8 | 12 | d_550088682_8
Deviation - Outside of tube   contaminated_8 | 13 | d_313097539_8
Date Received - Site   Shipment_9 |   | d_926457119_9
Object ID_10 | EDTA Tube 1 | d_825582494_10
Deviation_9 |   | d_678857215_9
Discard flag_9 |   | d_762124027_9
Object Collected_10 |   | d_593843561_10
Deviation - Not found_9 | 1 | d_982885431_9
Deviation - Unexpected tube   size or type_8 | 2 | d_777486216_8
Deviation- Low volume_9 | 3 | d_728366619_9
Deviation - Discard_8 | 4 | d_810960823_8
Deviation- Other_9 | 5 | d_453343022_9
Deviation- Improper   temperature - too high_9 | 6 | d_690540566_9
Deviation- Hemolyzed_8 | 7 | d_242307474_8
Deviation- Broken_9 | 8 | d_472864016_9
Deviation- Leaked/spilled_9 | 9 | d_757246707_9
Deviation - Mislabeled-   discard_9 | 10 | d_684617815_9
Deviation - Mislabeled-   Resolved_9 | 11 | d_283900611_9
Deviation-Improper temperature   - too low_9 | 12 | d_550088682_9
Deviation - Outside of tube   contaminated_9 | 13 | d_313097539_9
Date Received - Site   Shipment_10 |   | d_926457119_10
Deviation Details_4 |   | d_536710547_4
Object ID_11 | EDTA Tube 2 | d_825582494_11
Deviation_10 |   | d_678857215_10
Discard flag_10 |   | d_762124027_10
Object Collected_11 |   | d_593843561_11
Deviation - Not found_10 | 1 | d_982885431_10
Deviation - Unexpected tube   size or type_9 | 2 | d_777486216_9
Deviation - Discard_9 | 3 | d_810960823_9
Deviation- Low volume_10 | 4 | d_728366619_10
Deviation- Other_10 | 5 | d_453343022_10
Deviation- Improper   temperature - too high_10 | 6 | d_690540566_10
Deviation- Hemolyzed_9 | 7 | d_242307474_9
Deviation- Broken_10 | 8 | d_472864016_10
Deviation- Leaked/spilled_10 | 9 | d_757246707_10
Deviation - Mislabeled-   discard_10 | 10 | d_684617815_10
Deviation - Mislabeled-   Resolved_10 | 11 | d_283900611_10
Deviation-Improper temperature   - too low_10 | 12 | d_550088682_10
Deviation - Outside of tube   contaminated_10 | 13 | d_313097539_10
Date Received - Site   Shipment_11 |   | d_926457119_11
Object ID_12 | EDTA Tube 3 | d_825582494_12
Deviation_11 |   | d_678857215_11
Discard flag_11 |   | d_762124027_11
Object Collected_12 |   | d_593843561_12
Deviation - Not found_11 | 1 | d_982885431_11
Deviation - Unexpected tube   size or type_10 | 2 | d_777486216_10
Deviation - Discard_10 | 3 | d_810960823_10
Deviation- Low volume_11 | 4 | d_728366619_11
Deviation- Other_11 | 5 | d_453343022_11
Deviation- Hemolyzed_10 | 6 | d_242307474_10
Deviation- Improper   temperature - too high_11 | 7 | d_690540566_11
Deviation- Broken_11 | 8 | d_472864016_11
Deviation- Leaked/spilled_11 | 9 | d_757246707_11
Deviation - Mislabeled-   discard_11 | 10 | d_684617815_11
Deviation - Mislabeled-   Resolved_11 | 11 | d_283900611_11
Deviation-Improper temperature   - too low_11 | 12 | d_550088682_11
Deviation - Outside of tube   contaminated_11 | 13 | d_313097539_11
Date Received - Site   Shipment_12 |   | d_926457119_12

</body>

</html>

**Comment by FrogGirl1123** [comment]
This is a table of all possible default deviations
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/GERLAN~1/AppData/Local/Temp/1/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/GERLAN~1/AppData/Local/Temp/1/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



472864016 | Deviation-   Broken |  
-- | -- | --
102695484 | Deviation- Centrifuge -  did not clot at least 30 minutes |  
912088602 | Deviation- Centrifuge -  clotted for longer than 2 hours |  
861162895 | Deviation- Centrifuge - Improper   speed/force setting used (too high) |  
561005927 | Deviation- Centrifuge - Improper   speed/force setting used (too low) |  
654002184 | Deviation- Centrifuge - improper time   of spinning (too long) |  
937362785 | Deviation- Centrifuge - improper time   of spinning (too short) |  
635875253 | Deviation- Failed/broken gel layer |  
242307474 | Deviation- Hemolyzed |  
550088682 | Deviation-Improper temperature - too   low/frozen (record time and temperature Deviation in the comments) |  
690540566 | Deviation- Improper temperature - too   high (record time and temperature Deviation in the comments) |  
956345366 | Deviation - Insufficient volume - not   enough volume to transfer to urine tube (discard) |  
757246707 | Deviation- Leaked/spilled |  
728366619 | Deviation- Low volume -   (tube/container partially filled but still usable) |  
453343022 | Deviation- Other | first 15 start on row 373 and   end on row 402 of the data dictionary
810960823 | Deviation - Discard |  
777486216 | Deviation - Unexpected  tube size or type |  
684617815 | Deviation - Mislabeled- discard |  
283900611 | Deviation - Mislabeled- Resolved |  
982885431 | Deviation - Not Found |  
313097539 | Deviation - Outside of tube  contaminated |  
742806035 | Deviation - Participant used   mouthwash less than 30 seconds | the remaining ones start on row   801 and end on row 814



</body>

</html>

**Comment by FrogGirl1123** [comment]
@wyattks @StephanieWeinstein please let Dev Ops know if you would like any variation in the default deviations for the urine and mouthwash tubes other than the two issues that are specific to them.

**Comment by StephanieWeinstein** [comment]
Nicole,
There are deviations that are for the serum separator tubes only, that are not relevant to the other tubes. So we expected there would be more deviations on the serum separator tubes. These have to do with the centrifuging (the other tubes are not centrifuged)

102695484 | Deviation- Centrifuge -  did not clot at least 30 minutes |  
912088602 | Deviation- Centrifuge -  clotted for longer than 2 hours |  
861162895 | Deviation- Centrifuge - Improper   speed/force setting used (too high) |  
561005927 | Deviation- Centrifuge - Improper   speed/force setting used (too low) |  
654002184 | Deviation- Centrifuge - improper time   of spinning (too long) |  
937362785 | Deviation- Centrifuge - improper time   of spinning (too short) |  
635875253 | Deviation- Failed/broken gel layer

Does that explain the discrepancies you are finding?
Stephanie

**Comment by anthonypetersen** [comment]
I was going to say.. We wouldn't expect every deviation to be saved under ever tube, only the deviations that apply to each tube.

**Comment by FrogGirl1123** [comment]
@StephanieWeinstein  and @anthonypetersen We just got to that on a call with Michelle and Dominique.  The matrix that was given to the programmers was never given to my team so we don't have those distinctions made in the data dictionary.  Dominique is looking for the issue with the matrix to tag Madhuri and Jing in.  Could someone from the Biospecimen team please check the table above and confirm that you are seeing the correct defaults for each tube?

**Comment by StephanieWeinstein** [comment]
I can check for each tube that there is a number next to each deviation that goes with that tube, is that what you mean?  I can do this at 4pm.

**Comment by FrogGirl1123** [comment]
@StephanieWeinstein you'll have to look at column A to see read the deviations but column B will show which tube is associated with each object ID.  When I asked about if there were different deviations for different tubes in teams yesterday, this is what I meant.

**Comment by StephanieWeinstein** [comment]
Hi @FrogGirl1123, I reviewed the tables you have above and compared them with what we see in the dashboard entry screen and what we have listed in our SOPs for deviations.  Everything looks exactly right.  

But I am wondering if we want to add a simple "discard" option for urine and mouthwash like we have for blood, I have a question out to the biospecimen team on that one and will get back to you all on that.

and Nicole, sorry I misunderstood your question yesterday in Teams. That would have saved us (you) a lot of trouble.  The good news though is we are seeing what we expect so I don't think there needs to be any changes here (other than maybe adding the general "discard")

all- so we can talk in the morning about whether we need any wiping of Dev after all.  KP would like to get started entering participants into Dev, so if we can decide in the morning that would be great.

thank you!!
